src/containers/EmailInUse.js

```diff
@@ -0,0 +1,17 @@
+
+import { connect } from 'react-redux';
+
+import i18n from 'i18n-messages';
+import EmailInUse from 'components/EmailInUse';
+
+const mapStateToProps = (state, props) => {
+    return {
+        reset_url: state.main.dashboard_url + 'reset-password',
```

Comment:
I think it is better to decouple the reset password url from dashboard_url as the above url is now wrong.